server/services/placeOnReportService.ts

```diff
@@ -267,4 +274,26 @@ export default class PlaceOnReportService {
       })
     )
   }
+
+  async getPlaceholderValues(adjudicationNumber: number, user: User) {
+    const draftAdjudication = await this.getDraftAdjudicationDetails(adjudicationNumber, user)
+    const { prisonerNumber } = draftAdjudication.draftAdjudication
+    const associatedPrisonerNumber = draftAdjudication.draftAdjudication?.incidentRole?.associatedPrisonersNumber
+    const [prisonerDetails, associatedPrisoner] = await Promise.all([
+      this.getPrisonerDetails(prisonerNumber, user),
+      associatedPrisonerNumber && this.getPrisonerDetails(associatedPrisonerNumber, user),
+    ])
+    return {
+      offenderFirstName: properCaseName(prisonerDetails?.firstName),
+      offenderLastName: properCaseName(prisonerDetails?.lastName),
+      assistedFirstName: properCaseName(associatedPrisoner?.firstName),
+      assistedLastName: properCaseName(associatedPrisoner?.lastName),
+    }
+  }
+
+  async getPrisonerNumberFromDraftAdjudicationNumber(adjudicationNumber: number, user: User) {
```

Comment:
I am surprised this is needed - but prob is. The only problem is that it is quite expensive (getting the draft and prisoner info just for a prisoner number).

Reply:
Ah - it is needed to get back to the mainstream draft pages, which have the prisoner number